test/httpcrashtest.cpp

```diff
@@ -224,57 +214,54 @@ void HTTPCrashTest::testCrashForkit()
     const char* testname = "crashForkit ";
     try
     {
+        TST_LOG("Loading document");
         std::shared_ptr<http::WebSocketSession> socket
             = loadDocAndGetSession(_socketPoll, "empty.odt", _uri, testname);
 
+        TST_LOG("Allowing time for kit to connect to wsd to get cleanly killed");
+        std::this_thread::sleep_for(std::chrono::seconds(1));
+
         TST_LOG("Killing forkit.");
-        killForkitProcess();
-        TST_LOG("Communicating after kill.");
+        killPid(getForKitPid(), testname);
 
+        TST_LOG("Communicating after kill.");
         sendTextFrame(socket, "status", testname);
         assertResponseString(socket, "status:", testname);
 
         // respond close frame
         socket->asyncShutdown();
         LOK_ASSERT_MESSAGE("Expected successful disconnection of the WebSocket",
                            socket->waitForDisconnection(std::chrono::seconds(5)));
-
-        TST_LOG("Killing coolkit.");
-        killLoKitProcesses();
-        countCoolKitProcesses(0);
-        TST_LOG("Communicating after kill.");
-        socket = loadDocAndGetSession(_socketPoll, "empty.odt", _uri, testname);
```

Comment:
Where did this go ?

Reply:
As far as I can tell it doesn't do anything. The test is about killing forkit, it does that above and verifies we can still talk to the kit process.

But then it also kills the kit process, which shouldn't be necessary for the test.

Edit: Now I see it's probably verifying that forkit restarts and can spawn new kit processes. I'll re-add it.